Conventions fixes + no more home menu

PhoenixBound opened the pull request:
This makes things a bit more consistent, but I haven't tried Mocha or WUP Installer Mod to check some of the changes.

Some of these changes might have to be made to the 3DS guide as well, so I'll whip up another pull request for that.

PhoenixBound committed:
Grammar/spelling fixes

Not 100% sure about the "save editors and backup software" part

PhoenixBound committed:
Capitalization

SD might not need to be capitalized here. Also not sure if MOD or Mod is right.

PhoenixBound committed:
Remove home menu

Left one instance of "home menu" because it looks like it's the name of something in Mocha CFW itself.
"vWii Wii Menu" looks a little odd, but it sounds right.

PhoenixBound committed:
Merge branch 'master' into master

curtisy1 commented:
I agree to most of those, but "hombebrew" should actually be fixed(?) also vWii Wii menu sounds kind of repetitive so I would call it vWii menu only, but that's just my view on this. 

Still great job on finding some more typos! Keep it up as I won't be able to commit much from now on :)

PhoenixBound committed:
Extra fixes

vWii Menu instead of vWii Wii Menu
Also added a missing serial comma I noticed

PhoenixBound commented:
Fixed the part about vWii menu (also found another typo, so I fixed that too).

Could you elaborate about the part with "hombebrew"? I didn't catch what you meant by that.

curtisy1 commented:
Never mind, I actually fixed the typo in line 50 of get started.txt
(hombebrew) two days ago, but I just saw it was included again so sorry for
the confusion. Seeing this though, would you mind comparing the current
master with my previous pull requests and see if some mistakes were
reintroduced? Would really help me out if you did.

Thanks!

PhoenixBound committed:
Fix wupinstaller name

The name is pretty inconsistent, so I opted for names that are
relevant at the time of executing the steps.

PhoenixBound commented:
I think I found one possible error, but it's a little confusing and inconsistent so I'll just explain the whole thing here:

* The guide refers to wupinstaller as of this PR as *Modified WUP Installer* in Get-Started.txt and *WUP Installer Mod* in the other two pages.
* The release page for the tool calls it "WUP Installer y Mod {version}."
* The repository's readme calls it by its full original name, "WiiU public title installer."
* The HBL meta.xml file calls it "WUP Installer - Mod Y"
* And the app itself just calls it "WUP Installer" with a separate mod notice later.

One instance of "WUP Installer - Mod Y" was originally in the guide and taken out, which was how I noticed it at first.

For now, I'm just going to rename them to what they call themselves at the time they're being mentioned. Get-Started.txt will have "WUP Installer y Mod," and the other two pages will have "WUP Installer - Mod Y."

Plailect commented on _pages/en_US/Home.txt:
```diff
@@ -28,7 +28,7 @@ permalink: /index.html
 ## What is Homebrew?
 
-[**Homebrew**](https://en.wikipedia.org/wiki/List_of_homebrew_video_games) usually refers to software that is not authorized by Nintendo. It allows you to run homebrew games, tools like save editing and backup, and emulators for various older systems.
+[**Homebrew**](https://en.wikipedia.org/wiki/List_of_homebrew_video_games) usually refers to software that is not authorized by Nintendo. It allows you to run homebrew games, tools like save editors or backup software, and emulators for various older systems.
```
No, this is for actual save backups. This is NOT a reference to "backups".

Plailect commented on _pages/en_US/Mocha CFW (Haxchi).txt:
```diff
@@ -76,7 +76,7 @@ You can use the (Home) button to exit the Homebrew Launcher if you need to purch
 11. Press (A) to confirm and exit
 12. Your Wii U should display a black screen; this is the RedNAND setup
-13. Press the power button **on the console** (not the gamepad) to start the copy of your internal system memory to your SD card
+13. Press the power button **on the console** (not the gamepad) to start copying your internal system memory to your SD card
```
This change is incorrect.

PhoenixBound replied:
Whoops, I'll revert that then.

I'm a little confused about what it means to "start" a copy. Is it like "starting up"?

Plailect commented on _pages/en_US/Mocha CFW (Homebrew Launcher).txt:
```diff
@@ -35,7 +35,7 @@ In order to boot the patched signature checks and boot RedNAND, you will need to
 8. Press (A) to confirm and exit
 9. Your Wii U should display a black screen; this is the RedNAND setup
-10. Press the power button **on the console** (not the gamepad) to start the copy of your internal system memory to your SD card
+10. Press the power button **on the console** (not the gamepad) to start copying your internal system memory to your SD card
```
This change is incorrect.

Plailect commented on _pages/en_US/Mocha CFW (CBHC).txt:
```diff
@@ -4,7 +4,7 @@ lang: en
 ---
 
-CBHC (Coldboot Haxchi) changes your default system title to make your SysNAND launch the DS virtual console game by default instead of the home menu, which will then load RedNAND.
+CBHC (Coldboot Haxchi) changes your default system title to make your SysNAND launch the DS virtual console game by default instead of the Wii U Menu, which will then load RedNAND.
```
"home menu" is a much better descriptive term that is more easily identifiable

Plailect commented:
https://github.com/Plailect/Guide_WiiU/commit/4b0cee6abfa743a4876f881446b1922f7e53d65e
